main-project/src/boards/entity/board-bookmark.entity.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  BaseEntity,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { Board } from './board.entity';
+
+// fk없음, entity취합 후 생성예정
+@Entity()
+export class BoardBookmark extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  no: number;
+
+  @OneToOne((type) => Board, (board) => board.no)
+  @JoinColumn()
+  board: Board;
```

Comment:
```suggestion
  @OneToOne((type) => Board, (board) => board.no)
  @JoinColumn()
  boardNo: Board;
```
게시글 idx를 쓰는 거면 확실하게 No를 붙여주는 게 좋아요.

Reply:
오 이거는 저렇게도 되는군요 감사합니다